providers/fab/src/airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -116,7 +116,7 @@
         RESOURCE_ASSET_ALIAS,
     )
 else:
-    from airflow.providers.common.compat.security.permissions import (
+    from airflow.providers.fab.www.security.permissions import (
```

Comment:
The if branch and else branch are then the same (for this import), we can move it out from `if TYPE_CHECKING:` and add it to the list of line 70